From 23068b6ce78dab974825339649331e08f8195ea6 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Wed, 23 Dec 2015 17:16:02 +0200
Subject: [PATCH 3/7] arm: imx6ul: add u-boot-with-spl-cl.imx target for
 cl-som-imx6ul

Add u-boot-with-spl-cl.imx target for cl-som-imx6ul.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Makefile b/Makefile
index 2bc1421..0527298 100644
--- a/Makefile
+++ b/Makefile
@@ -968,6 +968,11 @@ SPL: spl/u-boot-spl.bin FORCE
 u-boot-with-spl.imx u-boot-with-nand-spl.imx: SPL u-boot.bin FORCE
 	$(Q)$(MAKE) $(build)=arch/arm/imx-common $@
 
+u-boot-with-spl-cl.imx: SPL u-boot.img FORCE
+	@dd if=SPL of=$@ bs=1K seek=0 conv=notrunc 2>/dev/null
+	@dd if=u-boot.img of=$@ bs=1K seek=63 conv=notrunc 2>/dev/null
+	@ln -sf $@ u-boot.imx
+
 MKIMAGEFLAGS_u-boot.ubl = -n $(UBL_CONFIG) -T ublimage -e $(CONFIG_SYS_TEXT_BASE)
 
 u-boot.ubl: u-boot-with-spl.bin FORCE
-- 
1.9.1

